<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoveCalculator</class>
 <widget class="QWidget" name="LoveCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 114, 147);</string>
  </property>
  <widget class="QLineEdit" name="firstPersonName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 182, 194);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondPersonName">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>301</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 182, 194);</string>
   </property>
  </widget>
  <widget class="QLabel" name="heading">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 40px;
color: rgb(81, 85, 51);</string>
   </property>
   <property name="text">
    <string>Love Calculator</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstPersonLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(118, 118, 118);</string>
   </property>
   <property name="text">
    <string>Enter First Person:</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondPersonLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(118, 118, 118);</string>
   </property>
   <property name="text">
    <string>Enter Second Person:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loveButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Love</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
font-family: Times New Roman;
color: rgb(100, 255, 200);


</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
